tools/CACertificates/certGen.sh

```diff
@@ -28,6 +28,15 @@ intermediate_ca_password="1234"
 root_ca_prefix="azure-iot-test-only.root.ca"
 intermediate_ca_prefix="azure-iot-test-only.intermediate"
 
+function makeCNsubject()
+{
+    local result="/CN=${1}"
+    case $OSTYPE in
+        msys|cygwin|win32) result="/${result}"
```

Comment:
Can you explain the purpose of this exactly? Does the script not work in cygwin environment without the preceding double slash?

Reply:
yeah, you right. double checked and remove cygwin from the expression